src/Azure.Functions.Cli/Kubernetes/KubernetesHelpers.cs

```diff
@@ -152,6 +161,8 @@ internal static string GetKedaResources(string @namespace)
             {
                 int position = 0;
                 var enabledFunctions = httpFunctions.ToDictionary(k => $"AzureFunctionsJobHost__functions__{position++}", v => v.Key);
+                //Add environment variables for the func app keys
+                FuncAppKeysHelper.AddAppKeysEnvironVariableNames(enabledFunctions, keysSecretCollectionName, mountKeysAsContainerVolume);
```

Comment:
It's hard to reason what this method actually doing just looking at the name, passed parameters, and their usage below.

I'd rather have this method return the dictionary of additional env var, then concat or merge the dictionaries when/before calling `GetDeployment`